### PR TITLE
grid-news & block-news

### PR DESCRIPTION
**Pull request by austinblanchard**

### CODE REVIEW

**Comment by drewbaker** [comment]
You also need to implement this: https://github.com/funkhaus/factory/issues/10#issuecomment-652603666

**Comment by drewbaker** [comment]
Also if you can describe in the PR description what component's this is for, that is good practice. Ideally you'd `#123` the related Component Request (which we don't have for these because I didn't do them here).

File: src/components/GridNews/GridNews.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    // Hovers
+    @media #{$has-hover} {
+        .block-news:hover {
+            .media {
+                transform: translateY(-40px);
+            }
+            .block-text {
+                transform: translateX(var(--unit-gutter));
+            }
+        }
```
[the remark this answers is not in the document]
Comment (austinblanchard) [reply src/components/GridNews/GridNews.vue:67]: Ah yeah, I had it in the block just forgot to delete here.

**Comment by drewbaker** [comment]
@austinblanchard Can you actually do a new PR with this addressed please:
https://github.com/funkhaus/factory/issues/10#issuecomment-652603666

**Pull request merged**